docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -180,10 +256,9 @@ Verify that all the files needed are here in the right folder, do the following
 
```

Comment:
On the line just above you do the ls -l in /usr/local/pf/conf/ssl/ instead of /usr/local/pf/conf/ssl/tls_certs